src/jobs/dto/update-job.dto.ts

```diff
@@ -1,4 +1,14 @@
-import { PartialType } from "@nestjs/swagger";
+import { ApiProperty,  PartialType } from "@nestjs/swagger";
 import { CreateJobDto } from "./create-job.dto";
+import { IsString } from "class-validator";
 
-export class UpdateJobDto extends PartialType(CreateJobDto) {}
+export class UpdateJobDto extends PartialType(CreateJobDto) {
+  @ApiProperty({
+    type: String,
+    required: true,
+    description:
+      "Id for the job to be updated.",
+  })
+  @IsString()
+  readonly id: string;
+}
```

Comment:
I'm not sure if this is correct and needed, as the update on job is not implemented. 
We are going to have statusUpdate. 
Check issue #622 for more info

Reply:
Fixed. I've removed job-update.dto entirely, and added the status update instead.